src/managers/GuildStickerManager.js

```diff
@@ -0,0 +1,161 @@
+   */
+
+  /**
+   * Resolves an StickerResolvable to a Sticker object.
+   * @method resolve
+   * @memberof GuildStickerManager
+   * @instance
+   * @param {StickerResolvable} sticker The Sticker resolvable to identify
+   * @returns {?Sticker}
+   */
+
+  /**
+   * Resolves an StickerResolvable to an Sticker id string.
+   * @method resolveId
+   * @memberof GuildStickerManager
+   * @instance
+   * @param {StickerResolvable} sticker The Sticker resolvable to identify
+   * @returns {?Snowflake}
+   */
+
+  /**
+   * Edits a sticker.
+   * @param {StickerResolvable} sticker The sticker to edit
+   * @param {GuildStickerEditData} [data] The new data for the sticker
+   * @param {string} [reason] Reason for editing this sticker
+   * @returns {Promise<Sticker>}
+   */
+  async edit(sticker, data, reason) {
+    const stickerId = this.resolveId(sticker);
+    if (!stickerId) throw new TypeError('INVALID_TYPE', 'sticker', 'StickerResolvable');
+
+    const d = await this.client.api.guilds(this.guild.id).stickers(stickerId).patch({
+      data,
+      reason,
+    });
+
+    const existing = this.cache.get(stickerId);
+    if (existing) {
+      const clone = existing._clone();
+      clone._patch(d);
+      return clone;
+    }
+    return this.add(d);
+  }
+
+  /**
+   * Deletes a sticker.
+   * @param {StickerResolvable} sticker The sticker to delete
+   * @param {string} [reason] Reason for deleting this sticker
+   * @returns {Promise<void>}
+   */
+  async delete(sticker, reason) {
+    sticker = this.resolveId(sticker);
+    if (!sticker) throw new TypeError('INVALID_TYPE', 'sticker', 'StickerResolvable');
+
+    await this.client.api.guilds(this.guild.id).stickers(sticker).delete({ reason });
+  }
+
+  /**
+   * Obtains one or more stickers from Discord, or the sticker cache if they're already available.
+   * @param {Snowflake} [id] The Sticker's id
+   * @param {BaseFetchOptions} [options] Additional options for this fetch
+   * @returns {Promise<Sticker|Collection<Snowflake, Sticker>>}
+   * @example
+   * // Fetch all stickers from the guild
+   * message.guild.stickers.fetch()
+   *   .then(stickers => console.log(`There are ${stickers.size} stickers.`))
+   *   .catch(console.error);
+   * @example
+   * // Fetch a single sticker
+   * message.guild.stickers.fetch('222078108977594368')
+   *   .then(sticker => console.log(`The sticker name is: ${sticker.name}`))
+   *   .catch(console.error);
+   */
+  async fetch(id, { cache = true, force = false } = {}) {
+    if (id) {
+      if (!force) {
+        const existing = this.cache.get(id);
+        if (existing) return existing;
+      }
+      const sticker = await this.client.api.guilds(this.guild.id).stickers(id).get();
+      return this.add(sticker, cache);
+    }
+
+    const data = await this.client.api.guilds(this.guild.id).stickers.get();
+    const stickers = new Collection(data.map(sticker => [sticker.id, this.add(sticker, cache)]));
```

Comment:
```suggestion
    return new Collection(data.map(sticker => [sticker.id, this.add(sticker, cache)]));
```
can't multiline comment on mobile, line below needs to be removed also